com.github.phase1geo.minder.yaml

```diff
@@ -0,0 +1,80 @@
+id: com.github.phase1geo.minder
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+base: io.elementary.BaseApp
+base-version: juno-19.08
+command: com.github.phase1geo.minder
+finish-args:
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/dconf
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=ca.desrt.dconf
+cleanup:
+  - '.la'
+  - '*.a'
+  - /cache
+  - /include
+  - /lib/pkgconfig
+  - /lib/systemd
+  - /lib/valadoc
+  - /lib/valadoc-0.48
+  - /man
+  - /share/aclocal
+  - /share/devhelp
+  - /share/gir-1.0
+  - /share/gtk-doc
+  - /share/man
+  - /share/vala-0.40
+  - /share/vala-0.48
+  - /share/valadoc
+  - /share/valadoc-0.48
+modules:
+  # We need new Vala for Minder > 1.6 version
+  - name: vala
+    cleanup-platform:
+      - '*'
+    config-opts:
+      - --enable-unversioned
+      - --enable-vapigen
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/vala/0.48/vala-0.48.6.tar.xz
+        sha256: d18d08ed030ce0e0f044f4c15c9df3c25b15beaf8700e45e43b736a6debf9707
+
+  - name: discount
+    buildsystem: simple
```

Comment:
Any reason for using a simple build here?

Reply:
Won't compile without this ld sed'ing stuff.